bigframes/ml/linear_model.py

```diff
@@ -155,14 +155,16 @@ def _fit(
     def predict(self, X: utils.ArrayType) -> bpd.DataFrame:
         if not self._bqml_model:
             raise RuntimeError("A model must be fitted before predict")
-
-        (X,) = utils.batch_convert_to_dataframe(X)
+        # add session
```

Comment:
Comments should explain "why", less so "what". We can remove this, as it's redundant with the line of code it's explaining.

```suggestion
```